### DISCUSSION TITLE
Squash multiple in one command

### DISCUSSION THREAD

**Discussion by johnwheeler** [jj-vcs/jj#2750] [Q&A] [answered]
As part of my workflow, I like to create a branch, begin creating commits on it, move the branch to my @- and do jj git push.

then I go and create a PR on GitHub and do a squash commit to merge it into my main branch.

I’d like to skip this PR creation step and do it from the CLI, but the squash command doesn’t work with revsets. Nor does the move command. Those are what I’ve tried.

Should I just fall back to using git rebase -I for this use case? Is that safe?

**Comment by martinvonz** [comment, marked as answer]
Yes, it's safe to mix `git` commands with `jj` commands.

We've wanted this feature for a while, and we recently created a tracking bug for it: #2678.

**Comment by wrrn** [comment]
This looks like it's been updated. We can now use `jj squash --from <rev-set>`.
